### PR TITLE
tweaks in support of courier queueing events to redis itself

### PR DESCRIPTION
**Pull request by nicpottier**

### CODE REVIEW

**Comment by nicpottier** [comment]
@rowanseymour can we merge and ship this so I can update courier?

File: temba/msgs/tasks.py [hunk 1]
```diff
@@ -287,9 +288,10 @@ def handle_event_task():
     messages that need to be handled.
 
     Currently three types of events may be "popped" from our queue:
-           msg - Which contains the id of the Msg to be processed
-          fire - Which contains the id of the EventFire that needs to be fired
-       timeout - Which contains a run that timed out and needs to be resumed
+           msg   - Which contains the id of the Msg to be processed
+          fire   - Which contains the id of the EventFire that needs to be fired
+       timeout   - Which contains a run that timed out and needs to be resumed
+    stop_contact - Which contains the contact id to stop
```
Comment (rowanseymour) [inline temba/msgs/tasks.py]: Lack of alignment here hurts :)

File: temba/msgs/models.py [hunk 2]
```diff
@@ -1189,7 +1189,7 @@ def queue_handling(self):
         # first push our msg on our contact's queue using our created date
         r = get_redis_connection('default')
         queue_time = self.sent_on if self.sent_on else timezone.now()
-        r.zadd(Msg.CONTACT_HANDLING_QUEUE % self.contact_id, datetime_to_ms(queue_time), dict_to_json(payload))
+        r.zadd(Msg.CONTACT_HANDLING_QUEUE % self.contact_id, datetime_to_s(queue_time), dict_to_json(payload))
```
Comment (rowanseymour) [inline temba/msgs/models.py]: What's the rationale for the switch from ms to s?

Comment (nicpottier) [reply temba/msgs/models.py]: Consistency.. we use seconds elsewhere.. also having epoch in MS makes no sense, this is a float, so if we are using epoch we should just use epoch.

File: temba/msgs/tasks.py (hunk 1 above)
Comment (nicpottier) [reply temba/msgs/tasks.py]: Did that just to mess with you.

**Comment by nicpottier** [comment]
@ericnewcomer mind checking this again for my own insurance / sanity?

**Pull request merged**